x-pack/metricbeat/docker-compose.yml

```diff
@@ -12,6 +12,16 @@ services:
     command: make
 
   # Modules
+  activemq:
+    image: docker.elastic.co/observability-ci/beats-integration-activemq:${ACTIVEMQ_VERSION:-5.15.9}-1
+    build:
+      context: ./module/activemq/_meta
+      args:
+        ACTIVEMQ_VERSION: ${ACTIVEMQ_VERSION:-5.15.9}
+    ports:
+      - 8161
+      - 61613
```

Comment:
is this port needed somewhere?

Reply:
8161 - REST endpoint, also Jolokia API exposed
61613 - STOMP, used by system tests to create queue and topic